[1.0.0, Git master] Link errors with ./configure --disable-static --disable-umockdev && make all check

This can be observed both _with_ umockdev installed and _without_. To reproduce:

```console
$ cd "$(mktemp -d)"
$ git clone --depth 1 https://github.com/USBGuard/usbguard
$ cd usbguard
$ ./autogen.sh
$ ./configure --disable-static --disable-umockdev && make all check
[..]
make[4]: Entering directory '/tmp/tmp.5lcNZbyNFE/usbguard/src/Tests'
  CXXLD    test-unit
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: Unit/test_unit-test_UEvent.o: in function `____C_A_T_C_H____T_E_S_T____0()':
test_UEvent.cpp:(.text+0x26): undefined reference to `usbguard::UEvent::UEvent()'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x1b2): undefined reference to `usbguard::UEvent::getAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x2e3): undefined reference to `usbguard::UEvent::getAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x479): undefined reference to `usbguard::UEvent::hasAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x588): undefined reference to `usbguard::UEvent::hasRequiredAttributes() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x66b): undefined reference to `usbguard::UEvent::clear()'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x88f): undefined reference to `usbguard::UEvent::setAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x9c0): undefined reference to `usbguard::UEvent::getAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0xb39): undefined reference to `usbguard::UEvent::hasAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0xc48): undefined reference to `usbguard::UEvent::hasRequiredAttributes() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0xd2b): undefined reference to `usbguard::UEvent::clear()'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0xf03): undefined reference to `usbguard::UEvent::getHeaderLine[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x1002): undefined reference to `usbguard::UEvent::toString[abi:cxx11](char) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: Unit/test_unit-test_UEvent.o: in function `____C_A_T_C_H____T_E_S_T____5()':
test_UEvent.cpp:(.text+0x1885): undefined reference to `usbguard::UEvent::UEvent()'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x1949): undefined reference to `usbguard::UEvent::setAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x1aa6): undefined reference to `usbguard::UEvent::setAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x1c03): undefined reference to `usbguard::UEvent::setAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x1e28): undefined reference to `usbguard::UEvent::getAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x1f59): undefined reference to `usbguard::UEvent::getAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x20ef): undefined reference to `usbguard::UEvent::hasAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x235c): undefined reference to `usbguard::UEvent::setAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x248d): undefined reference to `usbguard::UEvent::getAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x2606): undefined reference to `usbguard::UEvent::hasAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x280a): undefined reference to `usbguard::UEvent::hasRequiredAttributes() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x28f7): undefined reference to `usbguard::UEvent::getHeaderLine[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEvent.cpp:(.text+0x29cd): undefined reference to `usbguard::UEvent::toString[abi:cxx11](char) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: Unit/test_unit-test_UEventParser.o: in function `____C_A_T_C_H____T_E_S_T____0()':
test_UEventParser.cpp:(.text+0x6e): undefined reference to `usbguard::UEvent::UEvent()'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEventParser.cpp:(.text+0x10e): undefined reference to `usbguard::UEvent::fromString(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, bool, bool)'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEventParser.cpp:(.text+0x127): undefined reference to `usbguard::UEvent::operator=(usbguard::UEvent&&)'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEventParser.cpp:(.text+0x20c): undefined reference to `usbguard::UEvent::hasRequiredAttributes() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEventParser.cpp:(.text+0x32c): undefined reference to `usbguard::UEvent::getAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEventParser.cpp:(.text+0x4ac): undefined reference to `usbguard::UEvent::getAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEventParser.cpp:(.text+0x62c): undefined reference to `usbguard::UEvent::getAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEventParser.cpp:(.text+0x7ac): undefined reference to `usbguard::UEvent::getAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEventParser.cpp:(.text+0x92c): undefined reference to `usbguard::UEvent::getAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: Unit/test_unit-test_UEventParser.o:test_UEventParser.cpp:(.text+0xaac): more undefined references to `usbguard::UEvent::getAttribute(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const' follow
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: Unit/test_unit-test_UEventParser.o: in function `____C_A_T_C_H____T_E_S_T____0()':
test_UEventParser.cpp:(.text+0x1679): undefined reference to `usbguard::UEvent::getHeaderLine[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UEventParser.cpp:(.text+0x174f): undefined reference to `usbguard::UEvent::toString[abi:cxx11](char) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: Unit/test_unit-test_UMockdevDeviceDefinition.o: in function `____C_A_T_C_H____T_E_S_T____0()':
test_UMockdevDeviceDefinition.cpp:(.text+0x26): undefined reference to `usbguard::UMockdevDeviceDefinition::UMockdevDeviceDefinition()'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0x184): undefined reference to `usbguard::UMockdevDeviceDefinition::getUMockdevName[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0x2b4): undefined reference to `usbguard::UMockdevDeviceDefinition::getSysfsPath[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0x3e4): undefined reference to `usbguard::UMockdevDeviceDefinition::getDevfsNode[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0x514): undefined reference to `usbguard::UMockdevDeviceDefinition::getUMockdevDefinition[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0x643): undefined reference to `usbguard::UMockdevDeviceDefinition::hasDevfsNode() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0x91e): undefined reference to `usbguard::UMockdevDeviceDefinition::getUMockdevName[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0xa75): undefined reference to `usbguard::UMockdevDeviceDefinition::setUMockdevName(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0xb4b): undefined reference to `usbguard::UMockdevDeviceDefinition::getUMockdevName[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0xc6c): undefined reference to `usbguard::UMockdevDeviceDefinition::getSysfsPath[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0xdc3): undefined reference to `usbguard::UMockdevDeviceDefinition::setSysfsPath(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0xe99): undefined reference to `usbguard::UMockdevDeviceDefinition::getSysfsPath[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0xfba): undefined reference to `usbguard::UMockdevDeviceDefinition::getDevfsNode[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0x1111): undefined reference to `usbguard::UMockdevDeviceDefinition::setDevfsNode(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0x11e7): undefined reference to `usbguard::UMockdevDeviceDefinition::getDevfsNode[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0x12f9): undefined reference to `usbguard::UMockdevDeviceDefinition::hasDevfsNode() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0x13eb): undefined reference to `usbguard::UMockdevDeviceDefinition::getUMockdevDefinition[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0x1628): undefined reference to `usbguard::UMockdevDeviceDefinition::setUMockdevDefinition(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0x16fe): undefined reference to `usbguard::UMockdevDeviceDefinition::getUMockdevDefinition[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0x1a52): undefined reference to `usbguard::UMockdevDeviceDefinition::parseFromString(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, bool)'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0x1c6e): undefined reference to `usbguard::UMockdevDeviceDefinition::getUMockdevDefinition[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0x1c95): undefined reference to `usbguard::UMockdevDeviceDefinition::getUMockdevDefinition[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0x1e45): undefined reference to `usbguard::UMockdevDeviceDefinition::getSysfsPath[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: test_UMockdevDeviceDefinition.cpp:(.text+0x1f6c): undefined reference to `usbguard::UMockdevDeviceDefinition::getSysfsPath[abi:cxx11]() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: Unit/test_unit-test_UMockdevDeviceDefinition.o: in function `Catch::ExpressionLhs<usbguard::UMockdevDeviceDefinition const&>::endExpression()':
test_UMockdevDeviceDefinition.cpp:(.text._ZN5Catch13ExpressionLhsIRKN8usbguard24UMockdevDeviceDefinitionEE13endExpressionEv[_ZN5Catch13ExpressionLhsIRKN8usbguard24UMockdevDeviceDefinitionEE13endExpressionEv]+0x18): undefined reference to `usbguard::UMockdevDeviceDefinition::operator bool() const'
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: Unit/test_unit-test_UMockdevDeviceDefinition.o: in function `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > Catch::Detail::StringMakerBase<true>::convert<usbguard::UMockdevDeviceDefinition>(usbguard::UMockdevDeviceDefinition const&)':
test_UMockdevDeviceDefinition.cpp:(.text._ZN5Catch6Detail15StringMakerBaseILb1EE7convertIN8usbguard24UMockdevDeviceDefinitionEEENSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEERKT_[_ZN5Catch6Detail15StringMakerBaseILb1EE7convertIN8usbguard24UMockdevDeviceDefinitionEEENSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEERKT_]+0x43): undefined reference to `usbguard::UMockdevDeviceDefinition::operator bool() const'
collect2: error: ld returned 1 exit status
make[4]: *** [Makefile:861: test-unit] Error 1
make[4]: Leaving directory '/tmp/tmp.5lcNZbyNFE/usbguard/src/Tests'
make[3]: *** [Makefile:1566: check-am] Error 2
make[3]: Leaving directory '/tmp/tmp.5lcNZbyNFE/usbguard/src/Tests'
make[2]: *** [Makefile:1140: check-recursive] Error 1
make[2]: Leaving directory '/tmp/tmp.5lcNZbyNFE/usbguard/src/Tests'
make[1]: *** [Makefile:2715: check-recursive] Error 1
make[1]: Leaving directory '/tmp/tmp.5lcNZbyNFE/usbguard'
make: *** [Makefile:3508: check] Error 2
```

--- a/src/Tests/Unit/test_UEvent.cpp
+++ b/src/Tests/Unit/test_UEvent.cpp
@@ -17,7 +17,7 @@
 // Authors: Daniel Kopecek <dkopecek@redhat.com>
 //
 #include <catch.hpp>
-#include <UEvent.hpp>
+#include <UEvent.cpp>
 
 using namespace usbguard;
 

--- a/src/Tests/Unit/test_UEventParser.cpp
+++ b/src/Tests/Unit/test_UEventParser.cpp
@@ -17,7 +17,7 @@
 // Authors: Daniel Kopecek <dkopecek@redhat.com>
 //
 #include <catch.hpp>
-#include <UEvent.hpp>
+#include <UEventParser.cpp>
 
 using namespace usbguard;
 

--- a/src/Tests/Unit/test_UMockdevDeviceDefinition.cpp
+++ b/src/Tests/Unit/test_UMockdevDeviceDefinition.cpp
@@ -17,7 +17,7 @@
 // Authors: Daniel Kopecek <dkopecek@redhat.com>
 //
 #include <catch.hpp>
-#include <UMockdevDeviceDefinition.hpp>
+#include <UMockdevDeviceDefinition.cpp>
 
 #include "test_UMockdevDeviceDefinition.data.hpp"
 

